src/Uno.UI/UI/Xaml/Controls/Primitives/SelectorItem.cs

```diff
@@ -81,9 +81,12 @@ public SelectorItem()
 			// of a PointerPressed is a child that captures the pointer and may handle Pointer<Released/Exited|Canceled>
 			// without handling PointerPressed. In that case we wouldn't receive a PointerPressed without
 			// a matching Pointer<Released|Exited|Canceled> without handledEventsToo.
+			// Note: we also subscribe to PointerCaptureLostEvent to cleanup state also when pointer is canceled by scroller
+			//		 (i.e. with CanceledByDirectManipulation, in that case we only raise a CaptureLost event, not cancelled).
 			AddHandler(PointerReleasedEvent, _onPointerReleased, handledEventsToo: true);
 			AddHandler(PointerExitedEvent, _onPointerExitedOrCanceled, handledEventsToo: true);
 			AddHandler(PointerCanceledEvent, _onPointerExitedOrCanceled, handledEventsToo: true);
+			AddHandler(PointerCaptureLostEvent, _onPointerExitedOrCanceled, handledEventsToo: true);
```

Comment:
Additional tests needed to avoid SelectorItem remains in pressed state when srcolling kicks-in ?